apps/demo/tsconfig.json

```diff
@@ -22,6 +22,6 @@
       "react": ["./node_modules/@types/react"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts", "out/types/**/*.ts"],
```

Comment:
Why did we add this? maybe something was automatically added?

Reply:
It got automatically added. Should I remove it.

Reply:
I think yeah, bundled code shouldn't be type checked.

Reply:
Done